### ISSUE TITLE
Domain whitelisting not working

### ISSUE THREAD

**Issue by narcolepticinsomniac** [EFForg/privacybadgerchrome#610]
Privacy Badger started blocking all images from vk.me. The problem is that on any given page images are being pulled from a dozen different subdomains like: 
 (NSFW)
http://cs628520.vk.me/v628520704/14c41/bKD7V76L_Kw.jpg
or https://pp.vk.me/c624921/v624921684/3ef21/oHPpUuQh6_I.jpg
Every different page can have a dozen new subdomains. Whitelisting "vk.me" should allow them all but it does nothing.

**Comment by cooperq** [comment]
How are you whitelisting it? If you go to the options page and move the slider for vk.me from red to green that should work to whitelist all subdomains of vk.me, the second tab "whitelisted domains" is for disabling privacy badger when that domain is the *first party*. Does that make sense?

**Comment by narcolepticinsomniac** [comment]
I did think it would whitelist it as a third party also but I figured out that that isn't the case pretty quickly. That  really isn't the issue. I obviously didn't explain it clearly. There is no slider for "vk.me", just dozens of different subdomains and dozens of new ones on every different page. You can see what I mean on some pages on vk.com itself: http://vk.com/videos-62344632?section=all
The same method is used on lots of external sites who use vk as a free content host. I honestly searched for a while and couldn't find an example that was SFW.

**Comment by cooperq** [comment]
If you click on the gear icon in the privacy badger menu and go to the list of all domains you don't see an entry for vk.me? Can you double check to make sure? (try using ctrl+f to search for it).

**Comment by narcolepticinsomniac** [comment]
No, none in Devtools either. They seem to use it strictly to host images and the "sub" part of the domain is always a prefix so they are all treated as unique by PB. Pretty SFW example: http://vk.com/sashagrey.

**Comment by cooperq** [comment]
So if vk.me resources are getting blocked but vk.me is not showing up in the options page then this is a bug. Nothing should ever be getting blocked if it doesn't show up in the list of all domains in the options page. It is also strange that vk.me got blocked for you in teh first place. for me all of the vk.me domains are marked as 'not tracking'

**Comment by cooperq** [comment]
@SwartzCr should probably take a look at this.

**Comment by SwartzCr** [comment]
can you share a screenshot from a vk.com site where you see a slider for a thrid party vk.me site to be blocked? If vk.me isn't showing up in the options menu that's because it isn't setting cookies, which also means that it shouldn't be blocked. I'd be surprised if it was being blocked by privacy badger while not showing up on the options menu

**Comment by narcolepticinsomniac** [comment]
I'm happy to say I  can't provide any screenshots. Whitelisting "vk.me" appears to be working for all prefixed subdomains now. When I tested it immediately after, they were still blocked so I disabled PB. I just re-enabled it to take some screenshots for you and now it's working. I have an an extension set to clear history/cache/anything/everything on boot so maybe that had something to do with it. One way or another, it didn't work immediately but it does now.
